src/build-time-render/BuildTimeRender.ts

```diff
@@ -105,15 +118,19 @@ export default class BuildTimeRender {
 		html = html.replace(/href="(?!(http(s)?|\/))(.*?)"/g, `href="${prefix}$3"`);
 		html = html.replace(this._originalRoot, content);
 
-		const css = this._entries.reduce((css, entry) => {
+		let css = this._entries.reduce((css, entry) => {
 			const cssFile = this._manifest[entry.replace('.js', '.css')];
 			if (cssFile) {
 				html = html.replace(`<link href="${prefix}${cssFile}" rel="stylesheet">`, '');
-				css = `${css}<link rel="stylesheet" href="${prefix}${cssFile}" media="none" onload="if(media!='all')media='all'" />`;
+				css = `${css}<link rel="stylesheet" href="${prefix}${cssFile}" />`;
```

Comment:
Removed `media="none" onload="if(media!='all')media='all'"` to improve load performance.